apps/ws/src/shared/subscriber-online/subscriber-online.service.ts

```diff
@@ -30,7 +30,7 @@ export class SubscriberOnlineService {
 
   private async updateOnlineStatus(subscriber: ISubscriberJwt, updatePayload: IUpdateSubscriberPayload) {
     await this.subscriberRepository.update(
-      { _id: subscriber._id, _organizationId: subscriber.organizationId },
+      { _id: subscriber._id, _environmentId: subscriber.environmentId },
```

Comment:
Does anyone know if there is a reason to query by organization id here?
Updating it to environment id means that we can remove the _organizaitionId index from the DB.

Reply:
Let's remove it.